Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1

```diff
@@ -10641,7 +10641,7 @@ https://psappdeploytoolkit.com
             Return
         }
         ## Dispose of previous balloon
-        If ($script:notifyIcon) {
+        If (Test-Path -LiteralPath 'variable:notifyIcon') {
```

Comment:
Same question here

Reply:
Same answer here as well.

Reply:
I think it'd be useful to have a chat about same of these. If you're on Discord, maybe hit me up there?

Reply:
Do you have a link at all? The one on the website seemed dead

Reply:
🤦🏻https://discord.gg/winadmins